cabal.project

```diff
@@ -17,10 +17,10 @@ source-repository-package
   --sha256: sha256-dQVyvzVhVUOsLUuX8XQvN/wPGmbIvYeAmX7RpnT1gk8=
 
 index-state:
-  -- Bump this if you need newer packages from Hackage
-  , hackage.haskell.org 2023-11-15T00:00:00Z
+  -- Bump both the following dates if you need newer packages from Hackage
```

Comment:
This misleading. It is not requirement to update both of the indices to get newest packages from hackage.
```suggestion
  -- Bump this if you need newer packages from Hackage
```

Reply:
Fixed.